packages/dashboard/src/components/MultiCount/MultiCount.tsx

```diff
@@ -100,7 +104,7 @@ export class MultiCount extends React.Component<IMultiCountProps, IMultiCountSta
           onCardHide={this._hoverStateUpdate.bind(this, hoverKey, false)}
           onCardVisible={this._hoverStateUpdate.bind(this, hoverKey, true)}
         >
-          <div key={index} className={classNames.root} onClick={this._redirectToUrl.bind(this, href)}>
+          <div key={index} className={classNames.root} onClick={onClicked ? onClicked : this._redirectToUrl.bind(this, href)}>
```

Comment:
>onClick={onClicked ? onClicked : this._redirectToUrl.bind(this, href)}> [](start = 55, length = 71)

We will still need to get the href working, the click could only be for analytics.